lib/puppet/face/strings.rb

```diff
@@ -50,6 +50,7 @@ def check_required_features
       # For now, assume the remaining positional args are a list of manifest
       # and ruby files to parse.
       yard_args = (args.empty? ? MODULE_SOURCEFILES : args)
+      class YARD::Logger; def progress(*args); end; end
```

Comment:
:+1: Seems like a good idea to me! I use this line often (I have it written on a sticky note)